src/main/webapp/js/adminHome.es6

```diff
@@ -16,18 +16,41 @@
  * @param {String} status
  * @returns {String} a HTML row of action result table
  */
+ /**
+* Substring used to check if join link was created
```

Comment:
Let's not use `/** ... */` comment for non-documentation. `// comment` will do, especially if the line is short like this.